tests/wpt/mozilla/tests/mozilla/FileAPI/resource/file-submission.py

```diff
@@ -20,8 +20,8 @@ def main(request, response):
 
     boundary = content_type[1].strip("boundary=")
 
-    body = "--" + boundary + "\r\nContent-Disposition: form-data; name=\"file-input\"; filename=\"upload.txt\""
```

Comment:
Why does the filename go away?

Reply:
I shouldn't have committed this change. This was what I had to do to get the test to "pass" in Chrome, Safari, and Firefox. The reason the filename is empty, the contents are empty, and the Content-Type is application/octet-stream is because the test actually degrades to submitting a form with an empty file input in those browsers. This is because the test relies on a servo specific testing extension on the file input object called `selectFiles`, which doesn't work in the other browsers.

I'll push replace this commit with the proper changes to have this test working in Servo in a few moments.

Reply:
Ok. Dropped the commit that had the `filename` change. The only change in this file will be the addition of the `\r\n` now.